first_boot-displaymanager-20160805.json

```diff
@@ -0,0 +1,15 @@
+{
+  "properties": [],
+  "tags": [
+    "displaymanager"

```

Comment:
We might be able to help you if you point us to your local web instance where the needle you created is matched and no other is found.